integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/PetStoreTestCase.java

```diff
@@ -0,0 +1,230 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.cellery.integration.scenario.tests.petstore;
+import io.cellery.integration.scenario.tests.BaseTestCase;
+import io.cellery.integration.scenario.tests.Constants;
+import io.cellery.integration.scenario.tests.petstore.domain.Cart;
+import io.cellery.integration.scenario.tests.petstore.domain.Order;
+import io.cellery.integration.scenario.tests.petstore.domain.PetAccessory;
+import io.cellery.integration.scenario.tests.petstore.domain.User;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+import static io.github.bonigarcia.wdm.DriverManagerType.CHROME;
+
+/**
+ * This includes the test cases related to hello world web scenario.
+ */
+public class PetStoreTestCase extends BaseTestCase {
+    private static final String backEndInstanceName = "pet-be-inst";
+    private static final String backEndImageName = "pet-be-cell";
+    private static final String frontEndInstanceName = "pet-fe-inst";
+    private static final String frontEndImageName = "pet-fe-cell";
+    private static final String version = "latest";
+    private static final String link = "petStoreBackend:pet-be-inst";
+    private WebDriver webDriver;
+    private WebDriverWait wait;
+    private User alice;
+    private User bob;
+    private Cart cart;
+    private PetAccessory[] itemsOfAlice;
+    private Order order;
+
+    @BeforeClass
+    public void setup() {
```

Comment:
Shall we add method level comments as well?

Reply:
Added for all methods.